nix/README.md

```diff
@@ -0,0 +1,16 @@
+# Description
+
+This folder contains configuration for [Nix](https://nixos.org/), a purely functional package manager used by the Status app for its build process.
+
+# Configuration
+
+The main config file is [`nix/nix.conf`](/nix/nix.conf) and it's main purpose is defining the [binary caches](https://nixos.org/nix/manual/#ch-basic-package-mgmt) which allow download of packages to avoid having to compile them yourself locally.
```

Comment:
typo
```suggestion
The main config file is [`nix/nix.conf`](/nix/nix.conf) and its main purpose is defining the [binary caches](https://nixos.org/nix/manual/#ch-basic-package-mgmt) which allow download of packages to avoid having to compile them yourself locally.
```

Reply:
k